pkg/kubectl/cmd/create/create_deployment_test.go

```diff
@@ -142,7 +143,7 @@ func TestCreateDeploymentNoImage(t *testing.T) {
 	cmd.Flags().Set("output", "name")
 	options := &DeploymentOpts{
 		CreateSubcommandOptions: &CreateSubcommandOptions{
-			PrintFlags: NewPrintFlags("created", legacyscheme.Scheme),
+			PrintFlags: NewPrintFlags("created", scheme.Scheme),
```

Comment:
The change is good (if it works), but stop coupling your pulls together.

Reply:
At least put them in a separate commit, so it's clear it's not part of the flag move.